src/pages/module/ssl/edit/index.tsx

```diff
@@ -11,21 +11,37 @@ import {
   update as updateSSL,
 } from '@/services/ssl';
 import { useForm } from 'antd/es/form/util';
+import { router } from 'umi';
 
 const layout = {
-  wrapperCol: { span: 8 },
+  labelCol: {
+    span: 2,
+  },
+  wrapperCol: {
+    span: 8,
+  },
 };
 
-const Detail: React.FC = () => {
+const tailLayout = {
```

Comment:
What's tail layout?

Reply:
This means end of form, such as Submit and Cancel Button. (they need an offset for keep alignment)

Reply:
I see 😊